go/teams/teams.go

```diff
@@ -238,14 +243,21 @@ func (t *Team) ImplicitTeamDisplayName(ctx context.Context) (res keybase1.Implic
 			}
```

Comment:
Can you also take a lookt at line 222 above?
```
 		if !invite.UserActive {
 			// Active invite for inactive member, e.g. keybase-type invite for reset user.
 			continue
 		}
```
Looks significant but I don't know whether it's correct or not

Reply:
Hmm, yeah I think that is wrong. Why did you put that in?

Reply:
See this: https://github.com/keybase/client/commit/492737ab8cf051dc9eb9064296a82a20811e0e1b#diff-fcb3aebd54ff57abce1bf487058f99d0L406

AnnotateInvites used to skip inactive invites. I had to change it there, but I wanted to preserve how ImplicitTeamDisplayName worked.